src/modules/Packages.rb

```diff
@@ -134,6 +139,8 @@ def main
       @base_source_id = nil
 
       @old_packages_proposal = nil
+
+      PackageSystem.EnsureSourceInit
```

Comment:
Is ok to do the initialization here or must I move this call to the `find_providers` method?